scio-core/src/main/scala/com/spotify/scio/values/DoubleSCollectionFunctions.scala

```diff
@@ -39,22 +39,26 @@ class DoubleSCollectionFunctions(self: SCollection[Double]) {
   // def sum: SCollection[Double] = this.stats().map(_.sum)
 
   /** Compute the standard deviation of this SCollection's elements. */
-  def stdev: SCollection[Double] = self.transform(_.stats.map(_.stdev))
+  def stdev: SCollection[Double] = self.transform(_.stats.withName("StdDev").map(_.stdev))
```

Comment:
I did this up front so that we can have a test for naming transforms on `DoubleSCollectionFunctions`. Since they're composite transforms I wanted to make the internal name deterministic to make an assertion. https://github.com/spotify/scio/blob/playground/scio-test/src/test/scala/com/spotify/scio/values/NamedTransformTest.scala#L45

Maybe instead I can remove these, and just assert the transform name starts with the given name and ignore the inner transform name?